mindmaps-graql/src/test/java/io/mindmaps/graql/examples/ExamplesTest.java

```diff
@@ -37,7 +38,7 @@
 
     @Before
     public void setUp() {
-        MindmapsGraph graph = MindmapsTestGraphFactory.newEmptyGraph();
+        MindmapsGraph graph = Mindmaps.factory(Mindmaps.IN_MEMORY).getGraph(UUID.randomUUID().toString().replaceAll("-", "a"));
```

Comment:
Should I reintroduce `MindmapsTestFactory` then ? It feels odd since our own API now supports in memory-graphs ?